frontend/src/components/DatabaseMigrationHistoryPanel.vue

```diff
@@ -3,9 +3,9 @@
     <div
       class="flex flex-row items-center text-lg leading-6 font-medium text-main space-x-2"
     >
-      {{ $t("migration-history.self") }}
+      <span>{{ $t("migration-history.self") }}</span>
```

Comment:
Probably should provide tooltip

Reply:
`<NTooltip>` doesn't trigger on `<button disabled>`. I know it's a limitation of HTML accessibility by talking to the author of Naive-UI.
So I need more time working on this. WON'T push in this release.

Reply:
Can we use the existing tooltip-wrapper solution for now?

Reply:
Regretfully no.
I tried. But it looks weird because button has `disabled:opacity-50`. It makes all descendants of `<button>` transparent.
<img width="230" alt="image" src="https://user-images.githubusercontent.com/2749742/157011820-67de50e1-9419-4cd6-8508-0cd06353b7fa.png">
😢

A good news is I'm writing a `<BBButton>` to manage tooltip and disabled status together. It might be almost painless to alter `<button>` to the new `<BBButton>`. 🧐💪

Reply:
@tianzhou PTAL